laos_gggi/data_functions/rivers_damage.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+from pyprojroot import here
+import os
+from tqdm.notebook import tqdm
+import numpy as np
+import geopandas as gpd
+
+from laos_gggi.data_functions.rivers_data_loader import load_rivers_data
+from laos_gggi import load_emdat_data, load_shapefile
+from laos_gggi.const_vars import RIVERS_HYDRO_DAMAGE_FILENAME
+
+
+def create_hydro_rivers_damage():
+    data_path = here("data")
+    if not os.path.isfile(os.path.join(data_path, RIVERS_HYDRO_DAMAGE_FILENAME)):
+        big_rivers = load_rivers_data()
+        emdat = load_emdat_data()
+
+        # Load shapefiles
+        world = load_shapefile("world", repair_ISO_codes=True)
+        # laos = load_shapefile("laos")
+
+        damage_df = gpd.GeoDataFrame(
+            (
+                emdat["df_raw_filtered_adj"]
+                .query('disaster_class == "Hydrometereological"')[
+                    [
+                        "ISO",
+                        "End Year",
+                        "Latitude",
+                        "Longitude",
+                        "River Basin",
+                        "Total_Damage",
+                    ]
+                ]
+                .dropna(how="all", subset=["Latitude", "Longitude"])
+                .assign(
+                    geometry=lambda x: gpd.points_from_xy(x.Longitude, x.Latitude),
+                    year=lambda x: pd.to_datetime(x["End Year"], format="%Y"),
+                )
+                .drop(columns=["End Year"])
+                .replace({0.0: np.nan})
+            ),
+            crs=world.crs,
+        )
+
+        def get_distance_to_rivers(rivers, points):
```

Comment:
```suggestion
        def get_distance_to_rivers(rivers, points, crs="EPSG:3395"):
```

In case we want to change this. Use the `crs` variable inside `to_crs` (instead of hard-coding EPSG:3395